keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepositoryTests.kt

```diff
@@ -178,8 +178,8 @@ abstract class ArtifactRepositoryTests<T : ArtifactRepository> : JUnit5Minutests
       BuildMetadata(
         id = 1,
         uid = "1234",
-        startedAt = "yesterday",
-        completedAt = "today",
+        startedAt = "2020-11-24T04:44:04.000",
+        completedAt = "2020-11-25T03:04:02.259",
```

Comment:
That doesn't seem to be the format in actual metadata I've looked at in the DB. They should include a zone marker.

Reply:
yep, you are right! good catch